src/schemathesis/checks.py

```diff
@@ -48,11 +50,11 @@ def content_type_conformance(response: GenericResponse, result: TestResult) -> N
     )
 
 
-def response_schema_conformance(response: GenericResponse, result: TestResult) -> None:
+def response_schema_conformance(response: GenericResponse, case: "Case") -> None:
```

Comment:
It is a backward-incompatible change - should be explicitly noted in the changelog